src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/TimelineEventFactory.kt

```diff
@@ -132,6 +155,7 @@ class TimelineEventFactory {
     sourceReference: UUID,
     eventType: TimelineEventType,
     contextualInfo: String?,
+    correlationId: UUID = UUID.randomUUID(),
```

Comment:
Re: my question on correlationID cardinality above .... here in the factory method correlationID is not optional, so assuming this is the only way/place to create a timeline event I dont think it can ever be null can it?